categories_map.py

```diff
@@ -1713,7 +1713,60 @@
 #where cmssw_pkgX can be full or part of start of cmssw Package name
 # "jetmet": [ "SubSystem1", "SubSystem2/Package"]
 CMSSW_LABELS = {
+  "trk": [
+    "Alignment/APEEstimation",  
+    "Alignment/HIPAlignmentAlgorithm",
+    "Alignment/KalmanAlignmentAlgorithm",
+    "Alignment/Laser",
+    "Alignment/MillePedeAlignmentAlgorithm",
+    "Alignment/OfflineValidation",
+    "Alignment/ReferenceTrajectories",
+    "Alignment/SurveyAnalysis",
+    "Alignment/TrackerAlignment",
+    "Alignment/TwoBodyDecay",  
+    "AnalysisAlgos/SiStripClusterInfoProducer",  
+    "CalibFormats/SiPixel",
+    "CalibFormats/SiStrip",
+    "CalibTracker",
+    "Calibration/TkAlCaRecoProducers", 
+    "CommonTools/TrackerMap",
+    "CondCore/SiPhase2",
+    "CondCore/SiPixel",
+    "CondCore/SiStrip",
+    "CondFormats/SiPhase2",
+    "CondFormats/SiPixel",
+    "CondFormats/SiStrip"
+    "CondTools/SiPhase2",
+    "CondTools/SiPixel",
+    "CondTools/SiStrip",
+    "CUDADataFormats/SiPixel",
+    "CUDADataFormats/SiStrip",
+    "DPGAnalysis/SiStripTools",
+    "DataFormats/Phase2ITPixelCluster",
+    "DataFormats/Phase2Tracker",
+    "DataFormats/PixelMatchTrackReco",
+    "DataFormats/SiPixel",
+    "DataFormats/SiStrip",
```

Comment:
The `/SiStrip` and `/SiPixel` is repeated in many places.
It may be nice to have a possibility to match the package name based on just the package name with e.g. `/Package` to match (`startswith`) any subsystem? 
@smuzaffar do you think that would be practical or more problematic?

Reply:
in an earlier version I actually had `"DataFormats/Si"`, but fully specifying it looks more future proof.

Reply:
lets support regexp and bot converts all names to `.*(PackageName).*` which should allow you to use `/SiPixel` to match all packages with  `/SiPixel`

Reply:
does it require further change in the `cms-bot` code?
@smuzaffar let's merge this and revisit once it's available.

Reply:
bot is now ready to support regexp matched against file name.  If you want to update this PR then please go ahead and ping me when ready to go in

Reply:
I'd rather go as it is now.